WISE-Community/WISE-Client
# feat(Constraint): Component constraint authoring

## Changes

- Implemented component constraint authoring component
- Added constraint authoring component to all the component advanced authoring popups
- Created domain objects ValueAndText, RemovalCriteria, RemovalCriteriaParam

## Test

- Make sure component constraint authoring works for all the components
- Make sure the step constraint authoring still works like before

Closes #1137
